apps/src/templates/teacherDashboard/LoginTypePicker.jsx

```diff
@@ -123,6 +133,36 @@ class LoginTypePicker extends Component {
       <div style={style.container}>
         <Heading3 isRebranded>{title}</Heading3>
         <p>{i18n.addStudentsToSectionInstructionsUpdated()}</p>
+        {window.CPA_EXPERIENCE && (
+          <p>
+            <span
+              className="fa fa-exclamation-triangle"
+              aria-hidden="true"
+              style={style.warningIcon}
+            />
+            <span style={style.warningHeader}>
+              {i18n.addStudentsToSectionPermissionHeader() + ' '}
+            </span>
```

Comment:
From the styling, it looks like we are trying to emphasis this part of the sentence. Consider using `<em>` or `<strong>` so that screen readers know the intent.

Reply:
I agree.